packages/renderer/src/lib/preferences/PreferencesConnectionsEmptyRendering.svelte

```diff
@@ -0,0 +1,12 @@
+<script lang="ts">
+import Markdown from '../markdown/Markdown.svelte';
+
+export let message: string;
```

Comment:
```suggestion
export let message: string = '';
```
maybe to avoid to display undefined at some point